pwgen_test.go

```diff
@@ -60,4 +78,17 @@ func TestAlphaNumSymbols(t *testing.T) {
 			t.Errorf("Expected length %d, got %d\n", i, len(password))
 		}
 	}
+	if !verifyGeneratedStringValidity(password, alphaNumSymbols){
+		t.Errorf("Expected string to contain only {%s}, but got %s\n", alphaNumSymbols, password)
+	}
 }
+
+
+func verifyGeneratedStringValidity(generatedString string, format string)(bool){
```

Comment:
Maybe rename the function to somthing shorter like `verifyPassword()`?

Reply:
Would "verifyGeneratedPassword" make sense? Since its generated and I felt like the obligation of the method should be made very clear in method signature?

Reply:
Rename the method to "verifyGeneratedPassword" and also have run go fmt !

Reply:
Thanks!